packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/import_saved_objects.ts

```diff
@@ -170,6 +177,7 @@ export async function importSavedObjectsFromStream({
       type,
       id,
       meta,
+      managed: createdObjectManaged ?? managed,
```

Comment:
NIT: we're passing `managed` to `createSavedObjects`, so in theory all the created objects will have correct value set, right? So I think `managed: createdObjectManaged` would have the same effect?

Reply:
yea, it would.